app/forms/vehicle_form/remaining_payment_form.rb

```diff
@@ -17,7 +17,7 @@ def payments_remain?
     end
 
     def save
-      attributes[:payment_remaining] = 0 if valid? && !payments_remain?
+      attributes[:payment_remaining] = nil if valid? && !payments_remain?
```

Comment:
This is so we can just check if this value is nil when returning to the page so we know which radio should be selected, instead of having to check whether it's nil OR zero, I don't see any reason for saving a 0 value if they select 'No', but I may be wrong